veggieseasons/lib/widgets/search_bar.dart

```diff
@@ -39,6 +39,7 @@ class SearchBar extends StatelessWidget {
               child: CupertinoTextField(
                 controller: controller,
                 focusNode: focusNode,
+                decoration: BoxDecoration(),
```

Comment:
According to the documentation `decoration` is nullable. So:
```suggestion
                decoration: null,
```

Reply:
Thanks! I updated it now